lib/quadstore.ts

```diff
@@ -302,10 +301,16 @@ export class Quadstore implements Store {
         return indexBatch.put(index.getKey(importedNewQuad), value);
       }, quadBatch);
     }, batch);
-    await pFromCallback((cb) => { batch.write(cb); });
+    await this.writeBatch(batch, opts);
     return { type: ResultType.VOID };
   }
 
+  private async writeBatch(batch: AbstractChainedBatch, opts: BatchOpts) {
+    if (opts.preWrite != null)
```

Comment:
Considering we're using typescript, I think this can become `if (opts.preWrite)`. Also, I'm vehemently against `if`s without curly braces. One would think that there are other, better hills to die on but no! This is mine.

Reply:
Oops, I did spot that convention in your code. I'm the opposite! Maybe I just wish I was back in Python. No problem to change it, of course!